src/UI/Art/SpriteArtist/SpriteOptions.java

```diff
@@ -299,6 +310,7 @@ public int getSelectionNormalized()
     public void setSelectedOption(int value)
     {
       super.setSelectedOption(value - minOption);
+      storedValue = getSelectionNormalized();
```

Comment:
!!!

Reply:
"I see", said the blind man, as he took up his hammer and saw.